cgp-utils_config.md

```diff
@@ -0,0 +1,114 @@
+# Configuration
+
+The [cpg-utils library](https://github.com/populationgenomics/cpg-utils) ([pypi](https://pypi.org/project/cpg-utils/)) contains a streamlined config management tool. This config management is used by the seqr-loader pipeline, but is useful in projects at any scale.
```

Comment:
"This config management is used by the seqr-loader pipeline" would date pretty quickly. It is already used by other production workflows (e.g. large-cohort workflow, and under-the-construction-gatk-sv), as well as by some other analysis scripts we are running across the teams. Maybe rephrase 🤔